injectTranslations.py

```diff
@@ -11,47 +11,53 @@
 import sys
 import collections
 
+
 def loadLanguageDict(filename):
     codeToLang = collections.OrderedDict()
-    file = open(filename)
-    for line in file:
-        regex = re.compile("([a-z][a-z]|[a-z][a-z]-[A-Z][A-Z]|[a-z][a-z][a-z]|[a-z][a-z]-[A-Z][A-Z]-.*?)\\s*=\\s*(.*)")  # e.g. "de", "de-DE", "ast", "ca-ES-valencia"
-        match = regex.match(line)
-        if match:
-            codeToLang[match.group(1)] = match.group(2)
-    return codeToLang
+    with open(filename) as File:
+        for line in File:
+            regex = re.compile("([a-z][a-z]|[a-z][a-z]-[A-Z][A-Z]|[a-z][a-z][a-z]|[a-z][a-z]-[A-Z][A-Z]-.*?)\\s*=\\s*(.*)")  # e.g. "de", "de-DE", "ast", "ca-ES-valencia"
+            match = regex.match(line)
+            if match:
+                codeToLang[match.group(1)] = match.group(2)
+        return codeToLang
 
 if len(sys.argv) != 4:
-    sys.stderr.write("Usage: " + sys.argv[0] + " <translationLangCode> <englishFile> <translatedFile>\n")
+    sys.stderr.write("Usage: {} <translationLangCode> <englishFile> <translatedFile>\n".format(sys.argv[0]))
     sys.exit()
 
 translationLangCode = sys.argv[1]
 translationLangCodeShort = re.sub('_.*', '', translationLangCode)
 if translationLangCodeShort == "el":
```

Comment:
You can remove some duplication by

```
if translationLangCodeShort == "el":
    translationLangCode = "el_GR"
...
```

And it would be useful to move the file pattern to a CONSTANT at the top of the module where it will be seen quickly.